python/jupyter/src/widget.ts

```diff
@@ -18,6 +18,18 @@ const PlausibleTracker = Plausible({
 class ChemiscopeBaseView extends DOMWidgetView {
     protected visualizer?: DefaultVisualizer | StructureVisualizer | MapVisualizer;
     protected guid!: string;
+    protected warnings: Warnings = new Warnings();
+    protected static className: string = 'base-view';
+
+    public render(): void {
+        PlausibleTracker.trackPageview({
+            url: (location.pathname.split('/')[1] || '') + '/' + (this.className as string),
```

Comment:
Are you sure this works with overridden **static** data members? I would have use functions instead, but maybe this works as well? It feels a bit strange scoping-wise that parent classes have access to child classes static data

Reply:
well spotted.